src/Symfony/Component/Validator/Tests/Constraints/AbstractConstraintValidatorTest.php

```diff
@@ -244,7 +244,7 @@ protected function expectValidateValueAt($i, $propertyPath, $value, $constraints
 
     protected function assertNoViolation()
     {
-        $this->assertCount(0, $this->context->getViolations());
+        $this->assertSame(0, $violationsCount = count($this->context->getViolations()), sprintf('0 violation expected. Got %d.', $violationsCount));
```

Comment:
I don't understand what you mean. The output will be something like this:

```
Expected no violations.
Failed asserting that actual size 1 matches expected size 0.
```

Isn't that clear enough?